### PR TITLE
gh-pages: Add `playground` link to homepage.

### LINKED ISSUE #26550: Playground: Should we show the playground link `https://threejs.org/playground` in homepage `https://threejs.org`
### Description

Playground link `https://threejs.org/playground` is accessible, but customer does not know it in homepage `https://threejs.org` without the source code.

### Solution

Add the playground link `https://threejs.org/playground` to homepage `https://threejs.org`

### Alternatives

Add the playground link `https://threejs.org/playground` in `docs`.

### Additional context

Playground link `https://threejs.org/playground`  is accessible in homepage `https://threejs.org` by now.

### PR DESCRIPTION
**Pull request by linbingquan**
Fixed #26550 

**Description**

Add `playground` link to homepage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,6 +48,7 @@ <h2>Learn</h2>
 						<li><a href="docs/index.html#manual/introduction/Creating-a-scene">documentation</a></li>
 						<li><a href="examples/#webgl_animation_keyframes">examples</a></li>
 						<li><a href="editor/">editor</a></li>
+						<li><a href="playground/">playground</a></li>
```
Comment (Mugen87) [inline index.html:51]: Should this be `node playground`?

Comment (linbingquan) [reply index.html]: > Should this be `node playground`?

Done.

Comment (sunag) [reply index.html]: @Mugen87 Maybe `node editor`?

Comment (Mugen87) [reply index.html]: Yes, that's even better.

Comment (Mugen87) [reply index.html]: 330952bbc55016f2302ce595fb3c372b714a0f5f

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
Let's merge this so the node editor becomes more visible!

**Pull request merged**